include/erlcloud_ddb_streams.hrl

```diff
@@ -36,8 +36,24 @@
 -record(ddb_streams_describe_stream,
         {stream_description :: #ddb_streams_stream_description{}
         }).
+-record(ddb_streams_record_dynamodb,
```

Comment:
Oh, never mind, I misread the documentation for [Record](http://docs.aws.amazon.com/dynamodbstreams/latest/APIReference/API_Record.html) and confused it for [StreamRecord](http://docs.aws.amazon.com/dynamodbstreams/latest/APIReference/API_StreamRecord.html). Your change is correct.